src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -143,23 +157,8 @@ export class TemplateActionService {
         return;
       case "process_template":
         // HACK - create an embedded template processor service instance to process template programatically
-        const templateToProcess = await this.container.templateService.getTemplateByName(
-          args[0],
-          this.container.row
-        );
-        const processor = new TemplateProcessService(
-          this.container.templateService,
-          this.container.templateVariables,
-          this.container.templateTranslateService,
-          this.container.templateFieldService,
-          this.container.tourService,
-          this.container.router,
-          this.container.route,
-          this.container.templateNavService,
-          this.container.settingsService,
-          this.container.serverService,
-          this.analyticsService
-        );
+        const templateToProcess = await this.templateService.getTemplateByName(args[0]);
+        const processor = new TemplateProcessService(this.injector);
```

Comment:
this is an example of now only having to pass one arg to create a standalone template processor instead of 11 imported services

Reply:
Why is the row (this.container.row) no longer needed on line 160?

Reply:
Really good catch, we had approved but not merged the PR for the update to the template override system which caused a conflict and I must have accepted the wrong version. The reason we pass the row is to identify if the current template is already an override (and so not to re-override and result in infinite loops). 

I've tidied the code now to more explicitly accept the `is_override_target` arg instead of the general row, and included here. I _think_ the code would still have been fine as we are processing a template that is not actually rendered on the page (the workaround action that we created to handle processing outside of the templating system), and so there is no container and hence no risk of infinite loops (we're not rendering rows, instead just things more like processing startup scripts and calc sheets) - but still better to be explicit.